index.js

```diff
@@ -105,6 +105,13 @@ function extendContext(context, opts) {
         return this[_CONTEXT_SESSION];
       },
     },
+    regenerateSession: {
```

Comment:
I don't want to add too many properties in `ctx`, maybe we can move this method to `ctx.session.regenerate()`, and it should be an async function.